src/Workspaces/Core/Portable/Remote/RemoteFeatureOptions.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis.Experiments;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Remote
+{
+    internal static class RemoteFeatureOptions
+    {
+        private const string LocalRegistryPath = @"Roslyn\Features\Remote\";
+
+        /// <summary>
+        /// Global switch that determines of OOP can be used for a language feature. Exposed through
+        /// user visible switch to let people opt-in/out of this behavior.
+        /// </summary>
+        public static readonly Option<bool> OutOfProcessAllowed = new Option<bool>(
+            nameof(RemoteFeatureOptions), nameof(OutOfProcessAllowed), defaultValue: false,
+            storageLocations: new OptionStorageLocation[] {
+                new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(OutOfProcessAllowed)),
+                new RoamingProfileStorageLocation("TextEditor.Roslyn.Features.OutOfProcessAllowed")
```

Comment:
Removed roaming option.  As this is just an experimental flag (and will eventually not be exposed to users), it doesn't make sense to roam along with the real options users set.